src/BenchmarkDotNet/Attributes/Jobs/InProcessAttribute.cs

```diff
@@ -1,7 +1,9 @@
+using System;
 using BenchmarkDotNet.Jobs;
 
 namespace BenchmarkDotNet.Attributes
 {
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true)]
```

Comment:
I think that we don't need `AllowMultiple` for `InProcess` attribute: there are basically two options: log output and dont. IMHO having two of them at the same time makes no sense.
```suggestion
```